hooks/post_gen_project.py

```diff
@@ -90,5 +90,18 @@ def remove_unrequested_plugin_examples():
      git push -u origin main
 
 4. Read the README for more info: https://github.com/napari/cookiecutter-napari-plugin
+"""
+    )
+
+    print("""
+5. The following default URLs have been added to `setup.cfg`:
+
+    Bug Tracker = https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}/issues
+    Documentation = https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}/blob/master/README.md
```

Comment:
```suggestion
    Documentation = https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}/blob/main/README.md
```
This is tricky as I think GitHub recently switched its default from `master` to `main`. I might just put the main repo ... not sure

Reply:
You're right, I think

```https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}#README.md```
would be better e.g.

https://github.com/napari/napari#readme